lib/jsdom/living/helpers/form-controls.js

```diff
@@ -20,7 +20,10 @@ const whatwgURL = require("whatwg-url");
 
 const NodeList = require("../generated/NodeList");
 const { domSymbolTree } = require("../helpers/internal-constants");
+const { closest } = require("../helpers/traversal");
```

Comment:
It's just a small detail, but I noticed that the path on this line (and the older one above) goes out of the `helpers` directory and back in again.

Reply:
Oh, sure does. I think I just copied the pattern from the line above ;) I'll clean it up.